src/Microsoft.ML.Data/Evaluators/MulticlassClassifierEvaluator.cs

```diff
@@ -256,6 +256,8 @@ public Double MacroAvgAccuracy
                 {
                     get
                     {
+                        if (_numInstances == 0)
+                            return 0;
```

Comment:
>0 [](start = 35, length = 1)

maybe NaN?

Reply:
NaN means not a number but if instances are zero then isn't it accurate to say accuracy is zero? 

---
In reply to: [196921243](https://github.com/dotnet/machinelearning/pull/385#discussion_r196921243) [](ancestors = 196921243)